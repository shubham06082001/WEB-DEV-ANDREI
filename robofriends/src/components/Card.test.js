import { shallow, mount, register } from "enzyme";
import React from "react";
import Card from "./Card";

it("expect to render card components", () => {
  expect(shallow(<Card />)).toMatchSnapshot();
});


//to run npm test -- --coverage