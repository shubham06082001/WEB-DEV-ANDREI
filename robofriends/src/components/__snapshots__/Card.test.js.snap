// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render card components 1`] = `ShallowWrapper {}`;
